### DISCUSSION TITLE
Windows 7 Pro and characters with accent

### DISCUSSION THREAD

**Discussion by Numki02** [complexlogic/rsgain#124] [General]
Hello everyone,

For convenience, I use rsgain with a Windows 7 Pro service pack 1 in a virtual machine.
Until now I used version 3.1.1 x64 of rsgain without problem.
I am now trying to use later versions and am experiencing the following issues with accented file names.
Here are the messages displayed for processing the "é.wma" file:

V3.5.1
Creating a directory tree...
	1 directory found...
	Scanning the directory for files...
	Analysis of 'c:\users\titi\music\a\aa\é.wma'
	Unable to open input: invalid argument
	No files were scanned

V3.2 and V3.3
Creating a directory tree...
	1 directory found...
	Scanning the directory for files...
	Analysis of 'c:\users\titi\music\a\aa\é.wma'
Then pop-up window “rsgain has stopped working”

I tried to solve the problem with the "CHCP" command to no avail.
Is there a solution to my problem?

PS: I want to keep Windows 7 Pro which is perfect for my use.

Thank you for your answers

**Comment by complexlogic** [comment]
This is most likely due to the lack of Unicode support on Windows 7. If you rename the file, does it work?

**Reply by Numki02** [reply]
Hello complexlogic
> If you rename the file, does it work?

Yes, if I rename the file it works.
But I don't want to rename my files in my audio library.

> This is most likely due to the lack of Unicode support on Windows 7

Why does this work with version 3.1.1 ?

If there is no solution I will continue to use version 3.1.1. 
Too bad because later versions support several file types (mp3, flac...) in the same directory.

Thank you for your support
